DigitalLearningSolutions.Web/ViewModels/Login/LoginViewModel.cs

```diff
@@ -0,0 +1,26 @@
+﻿namespace DigitalLearningSolutions.Web.ViewModels.Login
+{
+    using System.ComponentModel.DataAnnotations;
+
+    public class LoginViewModel
+    {
+        public LoginViewModel()
+        {
+            Username = string.Empty;
+            Password = string.Empty;
+            RememberMe = false;
+        }
+
+        [Display(Order = -1)]
+        [Required(ErrorMessage = "Please enter a username to log in.")]
+        public string Username { get; set; }
```

Comment:
Given that we're doing null checks for this field in the controller, should it be a nullable string?